ckanext/dataset_subscriptions/templates/dataset-subscriptions_email_body.j2

```diff
@@ -1,8 +1,18 @@
-{% set num = activities|length %}
 {{ _("Here's a list of recently updated datasets you're following in DMS:") }}
 
-{% for activity in activities %}
-  {{ _("{count}. Dataset title: {title}").format(count=loop.index, title=activity.data.package.title) }}
+{% for activity in activities -%}
+  {% if activity.dataset_groups is defined -%}
+    {% set group_name = activity.dataset_groups[0].title + ' | ' -%}
+  {% else -%}
+    {% set group_name = "" -%}
+  {% endif -%}
+  {% if activity.dataset_extras is defined -%}
+    {% set program_area = activity.dataset_extras[0].value + ' | ' -%}
+  {% else -%}
+    {% set program_area = "" -%}
+  {% endif -%}
+  {{ _("{count}. {group_name}{program_area}{title}").format(count=loop.index, title=activity.data.package.title,
```

Comment:
there's no need to wrap this string in translation _(). It's user input so we most likely won't have translation strings for it.